### DISCUSSION TITLE
Report | Week 12

### DISCUSSION THREAD

**Discussion by LeonmAndreesen** [SE-TINF22B6/time2shine#66] [Show and tell]
Good day together, 

in today's blog we like you to show status q. We did some work on the website. The website now has a funtional highscore system. This allows to compare the players against each other.  Additionally we included a button to now implement a new game, at least for the highscore. Moreover currently we are changing the fonts of our BlackJack cards from dummy-pokemon to some real BlackJackcards. 
Furthermore we have now a little wiki. The user can look up all the rules for the game. You can even get some cool tricks for the game. 
Here you can see a sreenshot of the infrastructure of the database.
![image](https://github.com/SE-TINF22B6/time2shine/assets/123726672/5c4be0f7-6722-4526-9926-bfe618101d7c)

**Comment by BenjaminB109** [comment]
Dear time2shine Team,
good idea with the highscore system. I think it's good that the cards now have a proper design. Keep up the good work.

**Comment by JonasEbbinghaus25** [comment]
Dear time2shine,
It's great to see how you are continuously working on improving your web service. In addition to the new highscore functionalities, the wiki is also a great feature to help new players get to grips with the game rules so that they can beat the current highscore more quickly. I'm already looking forward to the next features.

**Comment by fehlend0** [comment]
It sounds like you've made some great progress! The addition of a functional highscore system and the implementation of real Blackjack cards are exciting updates. The inclusion of a wiki for game rules and tips is also a thoughtful touch. Keep up the good work!
